sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -90,6 +90,7 @@ export interface TracingContext {
 
 // @public
 export interface TracingSpan {
+    addEvent(name: string, attributes?: Record<string, unknown>, startTime?: Date): void;
```

Comment:
I will add this and other core-tracing changes separately - but I may need some time to work out the right API shape